src/cloud_db/cloud_db.py

```diff
@@ -117,19 +117,18 @@ def lookup_uid(self, uid: str):
         """
         logger.info(f'Looking up uid: {uid} in Supabase')
         result = None
-        order_type = None
+        nfc_type = None
         try:
             data = self.client.table("Order_Tag").select("*").eq("Tag_ID", uid).execute()
             if len(data.data) > 0:
                 order = self.client.table("Order_Tag_Group").select("Order_ID").eq("Group ID",data[0]["Order_Tag_Group_ID"]).execute()
                 result = order[0]
-                order_type = NFCType.ORDER
+                nfc_type = NFCType.ORDER
             else:
                 ID = self.client.table("Employee_Tag").select("*").eq("Tag_ID", uid).execute()
                 employee = self.client.table("Employee").select("Name").eq("Unit ID", ID[0]["Employee_Unit_ID"])
```

Comment:
It should be `employees`, `name`, `unifi_id`, and `employee_unit_id`.

Reply:
I would also get both `name` and `unifi_id` as well since we need to have the `unifi_id` later for inserting records.

Reply:
Finally, I would return an `NFCTag` object for this function, in which case I would rename it to something that makes more sense like `lookup_nfc_tag` or `get_nfc_tag`. Note that `NFCTag` may have to change since the `value` will no longer be consistently a string, but instead it should be something like a dictionary with it being renamed to `data`, so something like this:
`data: Dict[str, Any] = {}`